ui/uihandler.go

```diff
@@ -119,6 +119,16 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 	labels := r.FormValue("labels")
+	resolution, _ := strconv.ParseFloat(r.FormValue("r"), 64) // nolint: gas
```

Comment:
I should have asked about this section - we shouldn't try to read form value when no form is submitted